packages/checkout-storefront/src/components/AutoSaveAddressForm/AutoSaveAddressForm.tsx

```diff
@@ -4,23 +4,34 @@ import {
   useAddressForm,
   UseAddressFormProps,
 } from "@/checkout-storefront/components/AddressForm/useAddressForm";
+import { AddressTypeEnum } from "@/checkout-storefront/graphql";
+import { useCheckoutFormValidationTrigger } from "@/checkout-storefront/hooks/useCheckoutFormValidationTrigger";
 import { useFormDebouncedSubmit } from "@/checkout-storefront/hooks/useFormDebouncedSubmit";
-import React from "react";
 
 type AutoSaveAddressFormProps = UseAddressFormProps &
-  Omit<AddressFormProps, "formProps" | "defaultInputOptions" | "children">;
+  Omit<AddressFormProps, "formProps" | "defaultInputOptions" | "children"> & {
+    type: AddressTypeEnum;
+  };
 
 export const AutoSaveAddressForm: React.FC<AutoSaveAddressFormProps> = ({
   defaultValues,
+  type,
   onSubmit,
   ...addressFormRest
 }) => {
+  const isShippingForm = type === "SHIPPING";
+
   const { formProps, onSubmit: handleSubmit } = useAddressForm({
     defaultValues,
     onSubmit,
   });
   const { getValues } = formProps;
 
+  useCheckoutFormValidationTrigger({
+    formProps,
+    scope: isShippingForm ? "billingAddress" : "shippingAddress",
```

Comment:
Is this correct? looks like it should be the other way round 🤔

Reply:
It's not, changing